Sources/kwarchart/KwarChart.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  KwarChart.swift
+//  kwarchart
+//
+//  Created by Amiel Jireh Cordova on 5/10/21.
+//
+
+import SwiftUI
+
+@available(iOS 13.0, *)
+public struct KwarChart {
+    
+    public static func drawPieChart(pieChartType: PieChartType, pieSeries: [PieSeries], legendPosition: LegendPosition) -> PieChartPresentationView? {
+        var pieChart: PieChartView? = nil
+        
+        switch (pieChartType) {
+        case .NORMAL:
+            pieChart = PieChartView(dataModel: PieChartDataModel.init(dataModel: pieSeries))
```

Comment:
omit init can go straight 
```suggestion
            pieChart = PieChartView(dataModel: PieChartDataModel(dataModel: pieSeries))
```